### ISSUE TITLE
Feature request: Trajectory fits

### ISSUE THREAD

**Issue by LuckyMD** [scverse/anndata#121]
Hi,

I was just discussing the AnnData to R conversion that @flying-sheep is developing, and we noticed that there is no support for trajectory fits in AnnData. For conversions between R and python, it would be helpful to have a standard way to store e.g., spline fits in reduced dimensional representations that Slingshot produces.

One of the main shortcomings of scanpy compared to what is available in R is the limited availability of trajectory inference methods. DPT and PAGA are available, and neither use fits to visualize trajectories. To allow python users to better integrate with TI methods from R, it would be good to determine a standard for storing this data.

**Comment by falexwolf** [comment]
Right, we don't have this! I'd say it's not something that is urgently lacking (this is the first time it's requested), but it would be nice-to-have. Do you have a suggestion and would potentially go forward with a pull request? The main problem that I see is that implementing, say, a plotting function within Scanpy for a result obtained with Slingshot, is a bit strange. Hence, I wonder what one would do with the information if it was possible to carry over from R to anndata?

**Comment by falexwolf** [comment]
Thanks for the explanation! You're simplest idea makes sense, but you're alluding that a comprehensive solution seems like a lot of work. :) So, this is not a small project and we'd need someone to do this.

**Comment by LuckyMD** [comment]
Exactly... Unless you know exactly how model fits are canonically stored in python (if there is a canonical way like in R's `glm` objects), then it's a bigger project. Would this qualify as a Master's project for a computer science student? Maybe together with a comparison of TI methods like the Saelens review? Or is this more HiWi stuff?

**Comment by falexwolf** [comment]
Hm, no I don't know how this is canonically stored in python. `statsmodels` might have a solution, but I don't know it very well.

Yes, could be both, Hiwi or Master project, depending on how deep you go. There is definitely interesting things one can say about it.
